app/src/main/java/com/oracle/javafx/scenebuilder/app/welcomedialog/WelcomeDialogWindowController.java

```diff
@@ -215,17 +225,105 @@ private void openDocument() {
 
         handleOpen(paths);
     }
+    
+    protected static AlertDialog questionMissingFilesCleanup(Stage stage, List<String> missingFiles) {
+        String withPath = missingFiles.stream()
+                                      .collect(Collectors.joining(System.lineSeparator()));
+        AlertDialog question = new AlertDialog(stage);
+        question.setDefaultButtonID(ButtonID.CANCEL);
+        question.setShowDefaultButton(true);
+        question.setOKButtonTitle(I18N.getString("alert.welcome.file.not.found.okay"));
+        StringBuilder shortMessage = new StringBuilder();
+        if (missingFiles.size() > 1) {
+            shortMessage.append(I18N.getString("alert.welcome.files.not.found.question"));

```

Comment:
Same as in the other error dialog. We should be able to make this a single general case dialog, e.g. the following file(s) are missing...

Reply:
Thanks, I agree with that and will simplify the dialog accordingly.

Reply:
I've remove the condition and the customized dialog. There is now one dialog for n=1 and n>1.